stock_mts_mto_rule/tests/test_mto_mts_route.py

```diff
@@ -142,6 +142,38 @@ def test_rename_warehouse(self):
         self.warehouse.name = new_warehouse_name
         self.assertEqual(new_rule_name, self.warehouse.mts_mto_rule_id.name)
 
+    def test_standard_mto_mts_buy_route(self):
```

Comment:
Shouldn't be a tests for mrp?

Reply:
yes, but also I added the buy route, that's an important part from the base mto+mts, that it does not make the buy from the vendor